### ISSUE TITLE
Alignment of grid items

### ISSUE THREAD

**Issue by kksevta** [STRML/react-grid-layout#311]
div 'b' is displaying above a. Although b is having y value more than a
```
   <div key="b"  _grid={{ x: 0, y: 5, w: 3, h: 2 }}>b</div>
   <div key="a"  _grid={{ x: 0, y: 0, w: 1, h: 2 }}>a</div>
   <div key="c"  _grid={{ x: 7, y: 0, w: 1, h: 2 }}>c</div>
```

**Comment by mcottingham** [comment]
Do you have the `verticalCompact={false}` prop set on the RGL component?  I have found that this can happen when the default `verticalCompact` setting is used (true).

**Comment by kksevta** [comment]
I am using verticalCompact as true, but then verticalCompact is not proper.

**Comment by cp** [comment]
@kksevta I think this might actually be the same issue I ran into here https://github.com/STRML/react-grid-layout/pull/342. It's fixed in RGL 0.13.5.

**Comment by STRML** [comment]
Yep - this is fixed in #342 - thanks @cp.
